configs/platforms/sles-11-x86_64.rb

```diff
@@ -4,17 +4,17 @@
   plat.servicetype "sysv"
   plat.add_build_repository "http://osmirror.delivery.puppetlabs.net/sles-11-deps-x86_64/sles-11-deps-x86_64.repo"
   plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/yum/sles/11/x86_64/pl-build-tools-sles-11-x86_64.repo"
-  packages = [
-    "aaa_base",
-    "libbz2-devel",
-    "make",
-    "pkgconfig",
-    "pl-autotools",
-    "pl-cmake",
-    "pl-gcc",
-    "readline-devel",
-    "rsync",
-    "zlib-devel"
+  packages = %w[
+    aaa_base
+    libbz2-devel
+    make
+    pkgconfig
+    pl-autotools
+    pl-cmake
+    pl-gcc8
```

Comment:
Since this platform definition is shared by both agent-runtime-7.x and agent-runtime-main, I'm thinking it would be better to install `pl-gcc8` in https://github.com/puppetlabs/puppet-runtime/blob/531fb1fd6b6740d793669f0514ee54b8cb027d19/configs/components/runtime-agent.rb#L6-L10